projects/organization-management/src/app/components/user-budget-form/user-budget-form.component.spec.ts

```diff
@@ -1,52 +1,43 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
-import { FormBuilder, ReactiveFormsModule } from '@angular/forms';
-import { TranslateModule } from '@ngx-translate/core';
-import { MockComponent, MockDirective } from 'ng-mocks';
+import { FormGroup } from '@angular/forms';
+import { FormlyConfig, FormlyForm } from '@ngx-formly/core';
+import { MockComponent } from 'ng-mocks';
 import { of } from 'rxjs';
-import { instance, mock, when } from 'ts-mockito';
+import { anything, instance, mock, when } from 'ts-mockito';
 
 import { AppFacade } from 'ish-core/facades/app.facade';
 import { Locale } from 'ish-core/models/locale/locale.model';
-import { FormControlFeedbackComponent } from 'ish-shared/forms/components/form-control-feedback/form-control-feedback.component';
-import { InputComponent } from 'ish-shared/forms/components/input/input.component';
-import { ShowFormFeedbackDirective } from 'ish-shared/forms/directives/show-form-feedback.directive';
 
 import { UserBudgetFormComponent } from './user-budget-form.component';
 
 describe('User Budget Form Component', () => {
   let component: UserBudgetFormComponent;
   let fixture: ComponentFixture<UserBudgetFormComponent>;
   let element: HTMLElement;
-  let fb: FormBuilder;
   let appFacade: AppFacade;
+  let formlyConfig: FormlyConfig;
 
   beforeEach(async () => {
     appFacade = mock(AppFacade);
+    formlyConfig = mock(FormlyConfig);
 
     await TestBed.configureTestingModule({
-      imports: [ReactiveFormsModule, TranslateModule.forRoot()],
-      declarations: [
-        MockComponent(FormControlFeedbackComponent),
-        MockComponent(InputComponent),
-        MockDirective(ShowFormFeedbackDirective),
-        UserBudgetFormComponent,
+      declarations: [MockComponent(FormlyForm), UserBudgetFormComponent],
```

Comment:
Would be great if this test used the FormlyTestingModule instead of mocking FormlyForm